lm_eval/tasks/flores_101.py

```diff
@@ -63,6 +68,184 @@ def max_generation_length(self):
         return 512
 
 
+class Flores101MT_fewshot_wmt_fr2en(Flores101MT):
+    VERSION = 0
+    DATASET_PATH = "gsarti/flores_101"
+    DATASET_NAME = "all"
+
+    def __init__(
+        self,
+        data_dir: Optional[str] = None,
+        cache_dir: Optional[str] = None,
+        download_mode: Optional[str] = None,
+        prompt_template: Optional[promptsource.templates.Template] = None,
+        example_separator: Optional[str] = "\n###\n",
+        text_target_separator: Optional[str] = " ",
+        save_examples: Optional[bool] = True,
+    ):
+        """
+        Args:
+            save_examples (bool, optional, defaults to True):
+                Whether to save each example and corresponding model predictions
+                to an output `dict`.
+            > Few-shot prompting args
+            example_separator (str, optional, defaults to '\n###\n'):
+                The string that will be used to separate the few-shot examples
+                from the prompt example.
+                Default: '\n###\n'
+                    See Webson & Pavlick (2022) https://arxiv.org/pdf/2109.01247.pdf
+                    for justification of this separator.
+            text_target_separator (str, optional, defaults to ' '):
+                The string that will be used to separate the prompt example
+                from the target text.
+                NOTE: This is assumed to be some form of whitespace-only separation,
+                    e.g. "\n\n", "\t", "  ", etc. Otherwise, you should update
+                    the Task's `promptsource` template with the appropriate
+                    separator(s).
+                Example:
+                    Q: Where is the Eiffel Tower located? A:{text_target_separator}Paris
+        """
+        assert (
+            text_target_separator.isspace()
+        ), f"`text_target_separator` must be whitespace only. Got: `{text_target_separator}`"
+        super().__init__(data_dir, cache_dir, download_mode)
+        self.prompt_template = prompt_template
+        self.save_examples = save_examples
+        self.example_separator = example_separator
+        self.text_target_separator = text_target_separator
+        self.cache_dir = cache_dir
+        self.download_mode = download_mode
+
+    def fewshot_docs(self) -> datasets.Dataset:
+        """Returns a wmt dataset split"""
+        return datasets.load_dataset(
+            "wmt14",
+            "fr-en",
+            cache_dir=self.cache_dir,
+            download_mode=self.download_mode,
+        )["validation"]
+
+
+class Flores101MT_fewshot_fr2en(Flores101MT):
```

Comment:
Same here:  Add a 1 sentence class docstring describing diffs from the vanilla `Flores101MT` class that it's derived from.